package.json

```
@@ -4,28 +4,25 @@
   "private": true,
   "license": "AGPL-3.0-or-later",
   "scripts": {
-    "build": "webpack --mode production",
-    "build-ci": "node --max_old_space_size=4096 -r ts-node/register/transpile-only scripts/ci-build.ts",
+    "build": "gulp build",
+    "build-ci": "node --max_old_space_size=4096 node_modules/gulp/bin/gulp.js ci",
```

```suggestion
    "build-ci": "gulp ci",
```

The problem should not exist anymore.

---

> The problem should not exist anymore.

Why?